README.md

```diff
@@ -100,9 +100,7 @@ The **API endpoint** can be used to:
 - Configure the logging and metric system.
 - `[BETA]` Configure the data tree of the guest-facing metadata service. The
```

Comment:
Is this still `BETA`? @alexandruag

Reply:
I would still consider it so until it gets some more use :(